discover/deployment/articles/06-upgrading-to-liferay-7-1/03-running-the-upgrade-process/01-running-the-upgrade-tool.markdown

```diff
@@ -141,18 +141,28 @@ Each file's properties are described next.
 
 ### Configuring app-server.properties [](id=configuring-app-server-properties)
 
-Specify the following information to configure the app server on which @product-ver@ is installed: 
+Specify the following information to configure the app server on which 
+@product-ver@ is installed: 
 
-**dir:**  the application server directory *(required)*
+**dir:**  the absolute path of the application server directory. *(required)*
 
 **extra.lib.dirs:**  a comma delimited list of extra directories containing any
-binaries or resources to add to the  class path *(required)*
+binaries or resources to add to the class path. Use all absolute OR all relative
+paths. *(required)*
 
-**global.lib.dir:**  the application server's global library directory 
-*(required)*
+**global.lib.dir:**  the application server's global library directory. Use 
+absolute or relative path. *(required)*
```

Comment:
If you use the relative path, what is it relative to?

Reply:
I describe the relative path in the paragraph below. But I can see how it would be better to mention its relativity in the definition. I'll do that and let you know after I follow up with Alberto's suggestion. Thanks!